src/mon/OSDMonitor.cc

```diff
@@ -5404,7 +5435,7 @@ int OSDMonitor::get_crush_rule(const string &rule_name,
     ret = newcrush.get_rule_id(rule_name);
     if (ret != -ENOENT) {
       // found it, wait for it to be proposed
-      dout(20) << __func__ << ": rule " << rule_name
+      dout(20) << __func__ << " rule " << rule_name
```

Comment:
i am not sure why you are removing the ":"? is this part of the "__func__" change?